tests/test_sqlalchemy.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+import sqlalchemy.exc
+from sqlalchemy import MetaData
+from sqlalchemy.engine import Engine
+from sqlalchemy.sql.expression import TextClause
+
+
+def test_create_view(snowflake_engine: Engine):
+    """SQLalchemy internally calls `cursor.description`, which will fail if `CREATE VIEW` isn't handled."""
+    with snowflake_engine.connect() as conn:
+        conn.execute(TextClause("CREATE VIEW foo AS SELECT * FROM information_schema.databases"))
+
+        result = conn.execute("SELECT database_name FROM foo")
+        assert result
+        assert result.fetchall() == [("DB1",)]
+
+
+@pytest.mark.xfail(
+    reason="information_schema.columns is missing expected columns",
+    strict=True,
+    raises=sqlalchemy.exc.ProgrammingError,
+)
+def test_reflect(snowflake_engine: Engine):
```

Comment:
duckdb does return [column comments](https://github.com/duckdb/duckdb/pull/10372) using `select * from duckdb_columns()` but its not exposed in `information_schema.columns`. Probably worth raising a issue on the duckdb repo.

In the meantime though we can workaround it.  `information_schema.columns` is [transformed](https://github.com/DanCardin/fakesnow/blob/b8e38a6ecf28173c11713a8ab5ca6287c8114ebb/fakesnow/transforms.py#L356) to `information_schema._fs_columns_snowflake` defined [here](https://github.com/DanCardin/fakesnow/blob/b8e38a6ecf28173c11713a8ab5ca6287c8114ebb/fakesnow/info_schema.py#L35). This we could extend to join to `duckdb_columns()`.

Reply:
I had originally tried adding columns to this query but ran into issues, before I realized i didn't know if this was even the right query. I'll see if i can't follow that suggestion though. Thanks!